lib/pdf_generator.ex

```diff
@@ -153,6 +157,9 @@ defmodule PdfGenerator do
     end
   end
 
+  defp generate_filebase(nil), do: generate_filebase(Misc.Random.string)
+  defp generate_filebase(filename), do: Path.join(System.tmp_dir, Zarex.sanitize(filename))
```

Comment:
Please remove this dependency. I'd rather see a note in the readme, that a valid filename is assumed and point to `Zarex` if users want to have this sanitized. This is that we're deliberately changing user input – so the filename passed-in isn't guaranteed to to be the one written to.

I'd argue this could be a security feature, but I#d rather see this functional throwing an error that silently trying to fix the issue.